src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -261,3 +278,95 @@ To use the EKCO add-on to remove a node:
    The EKCO purge script For information about the EKCO purge script, see [Purge Nodes](/docs/add-ons/ekco#purge-nodes) in _EKCO Add-on_.
 
 1. Remove the node from the cluster.
+
+## Troubleshoot Node Removal
+
+This section includes information about troubleshooting issues with node removal in Rook Ceph clusters.
+
+### Rook Ceph v1.0.4 is Unhealthy with Mon Pod Pending
+
+#### Symptom
+
+After you remove a node from a Rook Ceph v1.0.4 cluster and you run `kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status`, you see that Ceph is in an unhealthy state where a Ceph monitor (mon) is down. 
+
+For example:
+
+```
+health: HEALTH_WARN
+        1/3 mons down, quorum a,c
```

Comment:
@paigecalvert can we add the part of that the service is out of quorum as well ?